buildutil

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env sh
 set -eu
-go mod vendor
+test -z "$(go env GOWORK)" && go mod vendor || go work vendor
```

Comment:
I am testing go work for simplifying major upgrades. Unfortunately `go mod vendor` complains and demands to use `go work vendor`. I tried to make it a oneliner to keep the file small and because we would to add this everywhere.

Reply:
I didn't know about `go work` until you mentioned it here. 
> I tried to make it a oneliner to keep the file small and because we would to add this everywhere.

Do we really? We'd only need this in projects that use `go work` and not everywhere, don't we?

Reply:
Yeah, see my test setup below. If I try to call `go mod vendor` on anywhere in the `projects` directory, Go throws an error at me.

![image](https://github.com/rebuy-de/rebuy-go-sdk/assets/1698599/e55b09fd-87e5-4eee-9e67-263dfd365a12)

Reply:
And to actually explain why I try to use `go work`: When I do a breaking change, I can immediately see the impact on those other projects without need for modifying to `go.mod` in each project.

So I would do a breaking change and then call my `hack/buildall.sh` hack, which is basically a for loop.